plugins/modules/files/iso_get_files.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Ansible Project
+# Copyright: (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_get_files
+short_description: Fetch files from ISO
+description:
+    - This module is used to get specific files from ISO.
+author:
+    - Yuhua Zou (@ZouYuhua) <zouy@vmware.com>
+requirements:
+- "pycdlib"
+- "python >= 2.7"
+version_added: '5.5.0'
+
+options:
+   iso:
+     description:
+     - This is the absolute paths of source ISO file.
+     - Will fail if specified ISO file does not exist on local machine.
+     - 'Note: With all ISO9660 levels from 1 to 3, all file names are restricted to uppercase letters, numbers and
+       underscores (_). File names are limited to 31 characters, directory nesting is limited to 8 levels, and path
+       names are limited to 255 characters.'
+     type: path
+     required: yes
+   get_files:
+     description:
+     - The absolute path with file name on ISO file.
+     type: list
+     elements: dict
+     required: yes
+     suboptions:
+       file_in_iso:
+         description:
+         - the path of file in ISO
+         type: path
+       file_local:
+         description:
+         - the local path
+         type: path
+'''
+
+EXAMPLES = r'''
+- name: "Get file from ISO"
+  community.general.iso_get_files:
+    iso: "/Users/zouy/Documents/ubuntu-22.04-desktop-amd64.iso"
+    get_files:
+      - file_in_iso: "/preseed/ubuntu.seed"
+        file_local: "/tmp/ubuntu.seed.org"
+      - file_in_iso: "/boot/grub/grub.cfg"
+        file_local: "/tmp/grub.cfg.org"
```

Comment:
This comment is a suggestion, rather than a request. I think life could be easier on the users if they could do something like:
```yaml
- name: "Get file from ISO"
  community.general.iso_get_files:
    iso: /Users/zouy/Documents/ubuntu-22.04-desktop-amd64.iso
    dest_dir: /tmp/iso_files
    get_files:
      - /preseed/ubuntu.seed
      - /boot/grub/grub.cfg
```
with the result being the retrieval of the files and storage at:
- `/tmp/iso_files/preseed/ubuntu.seed`
- `/tmp/iso_files/boot/grub/grub.cfg`

Reply:
Thanks. It's very helpful.